src/handlers/transfers/handler.js

```diff
@@ -86,48 +87,63 @@ const prepare = async (error, messages) => {
     ['success', 'fspId']
   ).startTimer()
   if (error) {
+    histTimerEnd({ success: false, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
     throw ErrorHandler.Factory.reformatFSPIOPError(error)
   }
   let message = {}
+  if (Array.isArray(messages)) {
+    message = messages[0]
+  } else {
+    message = messages
+  }
+  const parentSpanService = 'cl_transfer_prepare'
+  const contextFromMessage = EventSdk.Tracer.extractContextFromMessage(message.value)
+  const span = EventSdk.Tracer.createChildSpanFromContext(parentSpanService, contextFromMessage)
   try {
-    if (Array.isArray(messages)) {
-      message = messages[0]
-    } else {
-      message = messages
-    }
     const payload = decodePayload(message.value.content.payload)
     const headers = message.value.content.headers
     const action = message.value.metadata.event.action
     const transferId = payload.transferId
+    span.setTags({ transactionId: transferId })
+    await span.audit(message, EventSdk.AuditEventAction.start)
     const kafkaTopic = message.topic
     let consumer
+    await span.debug({
```

Comment:
Please comment out in the same approach (with **TODO** comment) as ml-api-adapter, e.g: https://github.com/mojaloop/ml-api-adapter/pull/190/commits/971c1f21e08a72d9a533aeb1d6909ffdcabe6b6e.

Please also do a search for any `span.debug` that need to be similarly updated.

Reply:
I have commented out as per ml-api-adapter, this was the only span.debug